CHANGES/8182.bugfix.rst

```diff
@@ -0,0 +1,9 @@
+Move file existence and type checks to ``FileResponse``,
+and fix it to handle permission errors -- by :user:`steverep`.
```

Comment:
```suggestion
and handle permission errors -- by :user:`steverep`.
```

Reply:
I don't like my grammar here either, but I was trying to draw a clear distinction between what was just moved from the static handler to the prepare method, versus what wasn't handled at all.  Again from #8507 the legacy setup clearer used to expect the `resolve()` call to handle this.

Reply:
Take another look.  I simplified the title part and added a sentence about the actual behavior changes.